src/Voron/Data/BTrees/Tree.Compressed.cs

```diff
@@ -47,7 +47,7 @@ private bool TryCompressPageNodes(Slice key, int len, TreePage page)
                     {
                         // we've just put a decompressed page to the cache however we aren't going to compress it
                         // need to invalidate it from the cache
-                        DecompressionsCache.Invalidate(page.PageNumber, DecompressionUsage.Write);
+                        DecompressionsCache?.Invalidate(page.PageNumber, DecompressionUsage.Write);
```

Comment:
Why did you need to use `?.` operator in all those places? If tree is compressed then we are guaranteed that `DecompressionsCache` isn't null